specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/payments.json

```diff
@@ -0,0 +1,711 @@
+      "properties": {
+        "paymentMethod": {
+          "$ref": "#/definitions/PaymentMethodProjectionProperties",
+          "description": "Projection of a payment method"
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethodLinksListResult": {
+      "description": "The list of payment method links.",
+      "properties": {
+        "nextLink": {
+          "description": "The link (url) to the next page of results.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "value": {
+          "description": "The list of payment method links.",
+          "items": {
+            "$ref": "#/definitions/PaymentMethodLink"
+          },
+          "readOnly": true,
+          "type": "array"
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethodLogo": {
+      "description": "Logo of payment method.",
+      "properties": {
+        "mimeType": {
+          "description": "MIME type of the logo.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "url": {
+          "description": "Public URL of image of the logo.",
+          "readOnly": true,
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "PaymentMethodProjectionProperties": {
+      "description": "The properties of a payment method projection.",
+      "properties": {
+        "accountHolderName": {
+          "description": "The account holder name for the payment method. This is only supported for payment methods with family CreditCard.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "displayName": {
+          "description": "The display name of the payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "expiration": {
+          "description": "The expiration month and year of the payment method. This is only supported for payment methods with family CreditCard.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "family": {
+          "description": "The family of payment method.",
+          "enum": [
+            "CreditCard",
+            "CheckWire"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "paymentMethodFamily"
+          }
+        },
+        "id": {
+          "description": "Id of payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "lastFourDigits": {
+          "description": "Last four digits of payment method.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "logos": {
+          "description": "The list of logos for the payment method.",
+          "items": {
+            "$ref": "#/definitions/PaymentMethodLogo"
+          },
+          "type": "array"
+        },
+        "status": {
+          "description": "Status of the payment method.",
+          "enum": [
+            "active",
```

Comment:
It can be helpful to have a fallback status like "unknown" for the possibility of additional status values being added in future versions.

Reply:
that's a very good idea. Thanks!